titus-api/src/main/java/com/netflix/titus/api/jobmanager/service/V3JobOperations.java

```diff
@@ -88,7 +87,12 @@
      */
     Completable recordTaskPlacement(String taskId, Function<Task, Task> changeFunction);
 
-    Observable<JobManagerEvent<?>> observeJobs();
+    default Observable<JobManagerEvent<?>> observeJobs() {
+        return observeJobs(alwaysTrue(), alwaysTrue());
+    }
+
+    Observable<JobManagerEvent<?>> observeJobs(Predicate<Pair<Job<?>, List<Task>>> jobsPredicate,
```

Comment:
For the `jobsPredicate` shouldn't we have only the `Job<?>` parameter?

Reply:
discussed in person, this is making it more consistent with the `findJobs` API, where you can filter jobs (job events in this case) with criteria on their associated tasks.